uber/templates/registration/merch.html

```diff
@@ -4,13 +4,41 @@
 
 {% include 'barcode_client.html' %}
 
+<style type="text/css">
+    #give .merch-list { color: red; }
+</style>
+
 <script type="text/javascript">
 
     var $shirtOpts = $('<select/>').attr('id', 'shirt');
     $.each({{ c.MERCH_SHIRT_OPTS|jsonize }}, function(i,size) {
         $shirtOpts.append($('<option/>').val(size[0]).text(size[1]));
     });
 
+    /**
+     * Given the list of items returned by Attendee.merch_list, return a nested
+     * list of those items for display when that attendee's merch is checked.
+     */
+    var createMerchList = function (message, merchItems) {
```

Comment:
This is where we use the ``Attendee.merch_items`` list which the backend now passes to format a nicely formatted nested list.